kaitai_utils.py

```diff
@@ -90,9 +115,13 @@ def _object_to_dict(self, instance: Any, path: str) -> Generator[Dict[str, Any],
 
         @param instance: object that needs parsing to dictionary
         @param path: string representing the jsonpath to the current node in the object tree
+        @param _debug: this parameter contains the offset at which data was read from the original input stream
         @return: dictionary containing parsed fields and their respective parsed values in a dictionary
         """
+        new_debug = None
```

Comment:
Can we think of a better name for `new_debug`? I guess it contains the `start` and `end` offsets of the parsed value, or something?

Reply:
Fixed!